CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Security - in case of vulnerabilities.
 - [x] Added new property to `SubscriptionChangePlan`: `quantity`
 - [x] Added new properties to `ApiTracking`: `relatedIds`, `requestHeaders`, `responseHeaders`
 
+### Removed
```

Comment:
should we up library version?

Reply:
These entity/service are not used at all (no API requests to that endpoint last 2.5 months). I think it doesn't make sense to release new version only because of it

Reply:
Note endpoints are available yet?

Reply:
They are already removed

Reply:
It means the old version doesn't work and will be better if we bump the library version.

Reply:
@iamdevice I've increased version